Gemfile

```diff
@@ -64,16 +64,21 @@ gem 'will_paginate-bootstrap'
 # gem 'capistrano-rails', group: :development
 
 group :development, :test do
+  gem 'mysql2' , '>= 0.3.18' , '<0.5'
```

Comment:
@ankitkataria We are using postgres in production,I would prefer it for tests as well.

Reply:
Locally, I was using postgres for it too, use pg is as easy as `gem 'pg'` in the `:test` group and making necessary changes in  `database.yml`. I'll add an option for the same in the upcoming prs.